### ISSUE TITLE
Training epoches

### ISSUE THREAD

**Issue by Jiejiegary** [yhu01/PT-MAP#9]
Thank you for nice code! 

May I know how many epoches you have trained model using rotation and S2M2_R, respectively?

**Comment by Jiejiegary** [comment]
I have trained the model using rotation for 400 epoches and S2M2_R for 200 epoches (without loading any pre-trained model), but I just got 73% and 82% on miniImageNet. The reported results are 82% and 88%. Is there anyone having the similar problem?

**Comment by m1nt07** [comment]
@Jiejiegary Have you solved the problem? I also met a similar problem. 
This is the result that I trained the model WideResNet28_10(S2M2_R) on miniImageNet for 70 epochs:'final accuracy found 81.37 +- 0.15'. However, I get 'final accuracy found 88.82 +- 0.13' when I used features @yhu01 provided in README.

**Comment by yhu01** [comment]
Just added the --train_aug option for training the backbone.

**Comment by m1nt07** [comment]
@yhu01 Thank you for your replying. I followed your tips, added the --train_aug option and trained my model again. The accuracy is 83.65 +- 0.15, still not reaching 88.82 +- 0.13. Is there anything else I missed?
